### ISSUE TITLE
URI can't have trailing slash

### ISSUE THREAD

**Issue by joernheissler** [wbond/asn1crypto#107]
```python3
#!/usr/bin/env python3

from asn1crypto.x509 import URI

uri = URI('http://www.example.net/')
uri2 = URI.load(uri.dump())
print(uri2.native)
```

This outputs `http://www.example.net`, i.e. the trailing slash was stripped.

I found this while loading a Certificate that's issued by letsencrypt and dumping it again. comparing the results. Their AIA has a trailing slash.

I'm not sure if there is some standard that says that URIs must not have trailing slashes. But even then, asn1crypto shouldn't strip it, in my opinion.

**Comment by wbond** [comment]
This was happening because of https://tools.ietf.org/html/rfc5280#section-7.4. Will be fixed shortly.

**Comment by wbond** [comment]
This should be resolved by 9e15efd73f1b7cfba73cb56b3188eb879480f77c
